postgresql.json

```
@@ -0,0 +1,34 @@
+{
+    "PostgreSQL":{
+        "containers":{
+            "postgresql":{
+                "image":"postgres:10.6",
+                "launch_order":1,
+                "ports":{
+                    "5432":{
+                        "description":"PostgreSQL port. Suggested default: 5432",
```

Change suggested default to fit with the default setting a few lines below.
```suggestion
                        "description":"PostgreSQL port. Suggested default: 5433",
```